libs/elasticsearch/langchain_elasticsearch/cache.py

```diff
@@ -186,3 +236,213 @@ def clear(self, **kwargs: Any) -> None:
             refresh=True,
             wait_for_completion=True,
         )
+
+
+class ElasticsearchEmbeddingsCache(
+    BaseStore[str, List[float]], ElasticsearchCacheIndexer
```

Comment:
Can we just make this a `BaseStore[str, bytes]` and therefore a `ByteStore` and then we can initialize with `CacheBackedEmbeddings.from_bytes_store` like the main docs say? You can still store the metadata, text_input and namespace, but the vector would be stored as [binary field type](https://www.elastic.co/guide/en/elasticsearch/reference/current/binary.html).

Reply:
done as ByteStore